src/features/fireWeather/components/DailyForecastDisplay.tsx

```diff
@@ -23,67 +30,67 @@ const useStyles = makeStyles({
   }
 })
 
-interface Props {
-  values: ModelValue[] | undefined
-}
-
-const DailyModelsDisplay = ({ values }: Props) => {
+const DailyForecastDisplay = (props: {
+  values: (ModelValue | NoonForecastValue)[] | undefined
+  testId: string
+  title: string
+}) => {
```

Comment:
Can you define the `Props` interface instead of doing it here?

Reply:
And do the object destructuring assignment like `({ values, testId, title}: Props)`